<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopWidget</class>
 <widget class="QDialog" name="ShopWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="cardIdEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>卡号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>班级：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classesEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="studentidEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="hallBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family:heiti;
	font: 75 10pt &quot;黑体&quot;;
height:20px;
width:60px;
color:white;
background-color:rgb(1, 70, 39);
border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string>食堂</string>
   </property>
  </widget>
  <widget class="QPushButton" name="storeBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family:heiti;
	font: 75 10pt &quot;黑体&quot;;
height:20px;
width:60px;
color:white;
background-color:rgb(1, 70, 39);
border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string>商店</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shopButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family:heiti;
	font: 75 10pt &quot;黑体&quot;;
height:20px;
width:60px;
color:white;
background-color:rgb(1, 70, 39);
border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string>消费</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>消费价格表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>请选择需要消费的地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;黑体&quot;;
font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>食堂单价10元/件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>商店单价15元/件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>点击确认消费</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="moneyEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>余额：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addmoneyBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family:heiti;
	font: 75 10pt &quot;黑体&quot;;
height:20px;
width:60px;
color:white;
background-color:rgb(1, 70, 39);
border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string>充值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>营业额情况</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>食堂：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商店：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hallEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="storeEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="qmoneyBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family:heiti;
	font: 75 10pt &quot;黑体&quot;;
height:20px;
width:60px;
color:white;
background-color:rgb(1, 70, 39);
border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string>更新营业额</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
